### PR TITLE
Add documentation for timescaledb.bgw_log_level

### LINKED ISSUE #2738: Add documentation for `timescaledb.bgw_log_level`
For troubleshooting background workers, we are adding the ability to set the log level for jobs and the scheduler by introducing a new variable `timescaledb.bgw_log_level` in timescale/timescaledb#6188.

### PR DESCRIPTION
**Pull request by mkindahl**
# Description

Documentation of the new GUC `timescaledb.bgw_log_level`.

It also fixes some missing "code quotes" for things like `pg_dump` and `pg_restore`.

# Links

Fixes #2738 

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: migrate/troubleshooting.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Troubleshooting known issues
 excerpt: Troubleshooting known issues in database migrations
```
Comment (jnidzwetzki) [inline migrate/troubleshooting.md:3]: This page is mostly about migrations ([Migrate your data to Timescale / Troubleshooting](https://docs.timescale.com/migrate/latest/troubleshooting/)). Maybe the notes about the jobs would be better placed in 
https://docs.timescale.com/use-timescale/latest/user-defined-actions/troubleshooting/.

File: migrate/troubleshooting.md [hunk 2]
```diff
@@ -219,4 +220,54 @@ Timescale instance or "merge" source databases to target schemas.
...
+| Custom job        | Execution started                                    |
+| Recompression job | Recompression job completed                          |
+| Reorder job       | Chunk reorder completed                              |
+| Reorder job       | Chunk reorder started                                |
+| Scheduler         | New jobs discovered and added to scheduled jobs list |
+| Scheduler         | Scheduling job for launch                            |
+
+### Debug level 2
+
+The amount of information printed at each level varies between jobs,
```
Comment (jnidzwetzki) [inline migrate/troubleshooting.md:266]: It should be clear, but we could mention, that `DEBUG2` includes the messages of `DEBUG1`

**Review by jnidzwetzki — APPROVED** [review]

File: migrate/troubleshooting.md (hunk 1 above)
Comment (mkindahl) [reply migrate/troubleshooting.md]: :man_facepalming: 

Moved. Kept the fixes to backquoting `pg_dump` and similar words.

File: migrate/troubleshooting.md (hunk 2 above)
Comment (mkindahl) [reply migrate/troubleshooting.md]: Added a sentence about that.

File: use-timescale/troubleshoot-timescaledb.md [hunk 3]
```diff
@@ -142,10 +142,62 @@ psql [your connect flags] -d your_timescale_db < dump_meta_data.sql > dumpfile.t
...
+This variable is set to the value of
+[`log_min_messages`][log_min_messages] by default, which typically is
+`WARNING`. If the value of [`log_min_messages`][log_min_messages] is
+changed in the configuration file, this value will be used instead for
+`timescaledb.bgw_log_level`.
+
+### Debug level 1
+
+The amount of information printed at each level varies between jobs,
+but the information printed at `DEBUG1` is currently shown below.
```
[the remark this answers is not in the document]
Comment (jnidzwetzki) [reply use-timescale/troubleshoot-timescaledb.md:170]: I would like to have logging of retry times/events. It's currently hard to understand why the scheduler don't pick up a job after a failure.

[the remark this answers is not in the document]
Comment (mkindahl) [reply use-timescale/troubleshoot-timescaledb.md:170]: Probably debug messages in general when the next job is scheduled, with information about the computation as well.

**Comment by mkindahl** [comment]
Merging, but we might want to consider changing the debug printouts in the code (and update the documentation after that to correctly reflect the situation).

**Pull request merged**